openshift/config/common/helm/babylon-config/templates/cross-cluster-backup.yaml

```diff
@@ -114,15 +139,19 @@ spec:
             - -xc
             - |
               set -e
-              DIR="/backup/$(date -u +%FT%TZ)"
-              mkdir $DIR
-              oc --kubeconfig=/secret/kubeconfig.yaml get anarchysubjects -n anarchy-operator -o yaml > $DIR/anarchysubjects.yaml
-              oc --kubeconfig=/secret/kubeconfig.yaml get anarchyactions -n anarchy-operator -o yaml > $DIR/anarchyactions.yaml
-              oc --kubeconfig=/secret/kubeconfig.yaml get resourcepools -n poolboy -o yaml > $DIR/resourcepools.yaml
-              oc --kubeconfig=/secret/kubeconfig.yaml get resourcehandles -n poolboy -o yaml > $DIR/resourcehandles.yaml
-              oc --kubeconfig=/secret/kubeconfig.yaml get resourceclaims --all-namespaces -o yaml > $DIR/resourceclaims.yaml
-              # Remove backup files older than 3 days
-              find /backup/ -mmin +$[3 * 24 * 60] -delete
+              rm -rf /backup/in-progress
+              mkdir /backup/in-progress
+              oc --kubeconfig=/secret/kubeconfig.yaml get anarchysubjects -n anarchy-operator -o yaml > /backup/in-progress/anarchysubjects.yaml
+              oc --kubeconfig=/secret/kubeconfig.yaml get anarchyactions -n anarchy-operator -o yaml > /backup/in-progress/anarchyactions.yaml
+              oc --kubeconfig=/secret/kubeconfig.yaml get resourcepools -n poolboy -o yaml > /backup/in-progress/resourcepools.yaml
+              oc --kubeconfig=/secret/kubeconfig.yaml get resourcehandles -n poolboy -o yaml > /backup/in-progress/resourcehandles.yaml
+              oc --kubeconfig=/secret/kubeconfig.yaml get resourceclaims --all-namespaces -o yaml > /backup/in-progress/resourceclaims.yaml
+              TIMESTAMP="$(date -u +%FT%TZ)"
+              mv /backup/in-progress "/backup/${TIMESTAMP}"
+              rm -f /backup/latest
+              ln -s "${TIMESTAMP}" /backup/latest
+              # Remove backup dirs older than 3 days
+              find /backup/ -maxdepth 1 -type d -regextype posix-egrep -regex '.*/[0-9]{4}-[0-9]{2}-[0-9]{2}T.*' -mmin +$[3 * 24 * 60] -exec rm -rf {} \; || :
```

Comment:
`$[]`  is deprecated for arithmetics  in favor of `$((  ))` 

I think both are Bash-isms, right ?
just in case i would use  `/bin/bash` instead of `/bin/sh` as the first argument of the command

Reply:
Switched the operator and switched to `/bin/bash`